e2e/README.md

```diff
@@ -1,34 +1,20 @@
 # E2E tests
 
-## Setup E2E server
-
-You might have to run the following command, to make sure Tiller has the right privileges.
-
-```shell script
-cat << EOF | kubectl apply -f - 
-kind: ServiceAccount
-apiVersion: v1
-metadata:
-  name: tiller
-  namespace: kube-system
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: tiller
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: cluster-admin
-subjects:
-  - kind: ServiceAccount
-    name: tiller
-    namespace: kube-system
-EOF
+In order to run it locally you can do the following
+```shell
+eval $(minikube -p minikube docker-env)
 ```
-```
-helm init --service-account tiller
 
-# or if you installed Helm2 using Brew
-/usr/local/opt/helm@2/bin/helm init --service-account tiller
+Then you need to build the binary and the image. Notice that the Dockerfile is currently multiarch.
+Depending on your environment the easiest thing to do can be to manually modify the Dockerfile adding
+manually the values for TARGETARCH and TARGETOS
```

Comment:
```suggestion
Then you need to build the binary and the image. Notice that  since the Dockerfile includes multiarch
support, you may need to set `DOCKER_BUILDKIT=1` when running `docker build` for the `TARGETARCH`
and `TARGETOS` args to be populated.
```

Reply:
(I guess you tried to warn against those not being populated? Setting `DOCKER_BUILDKIT=1` should fix that)